### PR TITLE
Changing stop/start behavior

### PR DESCRIPTION
**Pull request by mohanasudhan**
As we see numerous failures/issue when we stop/start master,
we decide to keep master running for better user experience.

Signed-off-by: Mohan Gandhi <mohgan@amazon.com>

Note: Let me know if we want to change "stop" -> "stop-compute" command?

### CODE REVIEW

File: docs/source/commands.rst [hunk 1]
```diff
@@ -67,24 +67,20 @@ optional arguments:
...
-server until it is stopped. If desired, one can optionally use the
-:code:`--compute-only argument` to leave the Master Server intact.
+Sets the Auto Scaling Group parameters to :code:`min/max/desired = 0/0/0`
 
-.. note:: A stopped cluster won't charge for EC2 usage but will still charge for
-   EBS usage and Elastic IP addresses. Each time you bring up an instance it
-charges you for an hour so bringing it up and down multiple times within an hour
-isn't reccomended. For more info see `Stop and Start Your Instance
-<https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Stop_Start.html>`_.
+.. note:: A stopped cluster will only stops complete-fleet, but Master server
```
Comment (rajachan) [inline docs/source/commands.rst]: Just so this statement isn't ambiguous, we should say stopping a cluster will only _terminate_ the compute fleet.

File: docs/source/commands.rst [hunk 2]
```diff
@@ -67,24 +67,20 @@ optional arguments:
...
+will be running for debugging/development. Hence you will be charged
+for master server even when you stop your cluster. we have seen numerous
+failures when we stop/start master back, so we decide to keep it running
+for better user experience. Do use "delete" option to avoid cost
+and recreate the cluster back when need.
```
Comment (rajachan) [inline docs/source/commands.rst]: Nit: need->needed.

File: docs/source/commands.rst [hunk 3]
```diff
@@ -67,24 +67,20 @@ optional arguments:
...
+Sets the Auto Scaling Group parameters to :code:`min/max/desired = 0/0/0`
 
-.. note:: A stopped cluster won't charge for EC2 usage but will still charge for
-   EBS usage and Elastic IP addresses. Each time you bring up an instance it
-charges you for an hour so bringing it up and down multiple times within an hour
-isn't reccomended. For more info see `Stop and Start Your Instance
-<https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Stop_Start.html>`_.
+.. note:: A stopped cluster will only stops complete-fleet, but Master server
+will be running for debugging/development. Hence you will be charged
+for master server even when you stop your cluster. we have seen numerous
```
Comment (rajachan) [inline docs/source/commands.rst]: Nit: Capitalize, we -> We

Comment (bwbarrett) [inline docs/source/commands.rst]: This doesn't sound super customer friendly.  How about (replacing everything after "we have"...)

Previous versions of CfnCluster stopped the master node after terminating the compute fleet.  Due to a number of challenges with the implementation of that feature, the current version only terminates the compute fleet.  The master will remain running.  To terminate all EC2 resources and avoid EC2 charges, consider deleting the cluster.

File: docs/source/commands.rst (hunk 1 above)
Comment (bwbarrett) [inline docs/source/commands.rst]: will only terminate the compute fleet.

Comment (mohanasudhan) [reply docs/source/commands.rst]: done

File: docs/source/commands.rst (hunk 3 above)
Comment (mohanasudhan) [reply docs/source/commands.rst]: done

File: docs/source/commands.rst (hunk 1 above)
Comment (mohanasudhan) [reply docs/source/commands.rst]: done

File: docs/source/commands.rst (hunk 3 above)
Comment (mohanasudhan) [reply docs/source/commands.rst]: Overridden by previous comments

File: docs/source/commands.rst (hunk 2 above)
Comment (mohanasudhan) [reply docs/source/commands.rst]: Overridden by previous comments

**Pull request merged**